Config-Files/C&C++/Windows/tasks.json

```diff
@@ -14,21 +14,21 @@
         "${workspaceFolder}\\jspwTest.cpp",
         "&&",
         "g++",
-        "jspwTest.cpp",
+        "${workspaceFolder}\\jspwTest.cpp",

```

Comment:
I think we can skip using the "${workspaceFolder}\\" in all places because by default this commands will be running on the workspace base directory. So adding workspace variables in most of the places in redundant. We might remove that as well. aslo all other config files follows the same rule.

Reply:
Thanks, addressed.